### DISCUSSION TITLE
Missing feature. Tweak visual settings per map

### DISCUSSION THREAD

**Discussion by rust2** [ppy/osu#30440] [Ideas]
Implement a missing feature. Tweak visual settings or audio settings per map as in osu! stable. Currently, any change is applied globally.

Also, it would be good if map loading pause if any of these menus hovered because it's annoying when I tweak something and map loads so I must go back to song menu then load map again and hope I will finish tweaking until it loads again

I'm talking abouth these guys:
![Tq7IIBjMNo](https://github.com/user-attachments/assets/0f81c5ff-39b6-4ff7-b57a-4c1d4781c73e)

**Comment by peppy** [comment]
For what it's worth, it won't disappear as long as you are moving your mouse on the controls.
